app/subscriber/src/features/my-minister/MyMinister.tsx

```diff
@@ -0,0 +1,68 @@
+import * as styled from './styled';
+
+import { DateFilter } from 'components/date-filter';
+import {
+  IContentListAdvancedFilter,
+  IContentListFilter,
+} from 'features/content/list-view/interfaces';
+import { columns } from 'features/home/constants';
+import { makeFilter } from 'features/home/utils';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useContent } from 'store/hooks';
+import { ContentStatus, ContentTypeName, FlexboxTable, IContentModel, Page, Row } from 'tno-core';
+
+export const MyMinister: React.FC = () => {
+  const [{ filter, filterAdvanced }, { findContent }] = useContent();
+  const [homeItems, setHomeItems] = React.useState<IContentModel[]>([]);
+  const navigate = useNavigate();
+
+  const [, setLoading] = React.useState(false);
+  const fetch = React.useCallback(
+    async (filter: IContentListFilter & Partial<IContentListAdvancedFilter>) => {
+      try {
+        setLoading(true);
+        const data = await findContent(
+          makeFilter({
+            ...filter,
+            startDate: '',
+            endDate: '',
+            keyword: localStorage.getItem('myMinister') ?? '',
```

Comment:
Would we want to store this on a new column in the users table instead?

Reply:
We could add a `User.Preferences` property which would be JSONP like many of our configuration tables.

Reply:
Sounds like a good idea, will do!